packages/react-searchbox/examples/with-controlled-props/src/index.js

```diff
@@ -10,7 +10,7 @@ ReactDOM.render(
     url="https://appbase-demo-ansible-abxiydt-arc.searchbase.io"
     appbaseConfig={{
       recordAnalytics: true,
-      enableQueryRules: true,
+      enableQueryRules: false,
```

Comment:
Any reason to change it?

Reply:
to switch off getting promoted results back from the backend